README.md

```diff
@@ -27,17 +34,23 @@ Don't forget to use a strong, randomly generated password instead of `<PASSWORD>
 * Fill out the application settings (see [configuration section](#configuration))
 * Click on "Deploy"
 
+#### VPC configuration
```

Comment:
Should this be `###`?

Reply:
I indented it on purpose as a sub-chapter of the setup.